src/solver/src/grins_solver.C

```diff
@@ -52,6 +52,7 @@ namespace GRINS
       _max_linear_iterations( input("linear-nonlinear-solver/max_linear_iterations", 500 ) ),
       _continue_after_backtrack_failure( input("linear-nonlinear-solver/continue_after_backtrack_failure", false ) ),
       _continue_after_max_iterations( input("linear-nonlinear-solver/continue_after_max_iterations", false ) ),
+      require_residual_reduction( input("linear-nonlinear-solver/require_residual_reduction", false ) ),
```

Comment:
Did you try compiling this because I don't see how this could compile... This should be `_require_residual_reduction` (`_` prefix)